### PR TITLE
Ddcnl 3975

### PR DESCRIPTION
**Pull request by Muhammad-Ali-Ahmad**

### CODE REVIEW

File: test/uk/gov/hmrc/tai/forms/employments/UpdateRemoveEmploymentFormSpec.scala [hunk 1]
```diff
@@ -44,11 +44,11 @@ class UpdateRemoveEmploymentFormSpec extends BaseSpec with FormValuesConstants {
       val invalidChoice = Json.obj(choice -> "")
       val invalidatedForm = form.bind(invalidChoice)
 
-      invalidatedForm.errors.head.messages mustBe List(Messages("tai.error.chooseOneOption"))
+      invalidatedForm.errors.head.messages mustBe List(Messages("tai.employment.decision.error.empty", "HMRC"))
```
Comment (struong) [inline test/uk/gov/hmrc/tai/forms/employments/UpdateRemoveEmploymentFormSpec.scala:47]: use a test name, not a real company

File: test/uk/gov/hmrc/tai/forms/employments/UpdateRemoveEmploymentFormSpec.scala [hunk 2]
```diff
@@ -44,11 +44,11 @@ class UpdateRemoveEmploymentFormSpec extends BaseSpec with FormValuesConstants {
...
       invalidatedForm.value mustBe None
     }
   }
 
   val choice: String = UpdateRemoveEmploymentForm.EmploymentDecision
-  private val form = UpdateRemoveEmploymentForm.form
+  private val form = UpdateRemoveEmploymentForm.form("HMRC")
```
Comment (struong) [inline test/uk/gov/hmrc/tai/forms/employments/UpdateRemoveEmploymentFormSpec.scala:53]: use a test name, not a real company

File: test/views/html/employments/UpdateRemoveEmploymentDecisionViewSpec.scala [hunk 3]
```diff
@@ -26,7 +26,7 @@ class UpdateRemoveEmploymentDecisionViewSpec extends TaiViewSpec with FormValues
   val employmentName = "Employment Name"
   val empId = 1
   val updateRemoveEmploymentForm: Form[Option[String]] =
-    UpdateRemoveEmploymentForm.form.bind(Map(UpdateRemoveEmploymentForm.EmploymentDecision -> YesValue))
+    UpdateRemoveEmploymentForm.form("HMRC").bind(Map(UpdateRemoveEmploymentForm.EmploymentDecision -> YesValue))
```
Comment (struong) [inline test/views/html/employments/UpdateRemoveEmploymentDecisionViewSpec.scala:29]: use a test name, not a real company

File: app/controllers/employments/EndEmploymentController.scala [hunk 4]
```diff
@@ -89,7 +89,7 @@ class EndEmploymentController @Inject()(
         case Right(mandatoryValues) =>
           Ok(
             update_remove_employment_decision(
-              UpdateRemoveEmploymentForm.form,
+              UpdateRemoveEmploymentForm.form(mandatoryValues(0)),
```
Comment (struong) [inline app/controllers/employments/EndEmploymentController.scala:92]: Can you get through to this form if no employment was ever entered?

File: test/uk/gov/hmrc/tai/forms/employments/UpdateRemoveEmploymentFormSpec.scala (hunk 1 above)
Comment (Muhammad-Ali-Ahmad) [reply test/uk/gov/hmrc/tai/forms/employments/UpdateRemoveEmploymentFormSpec.scala]: done

File: test/uk/gov/hmrc/tai/forms/employments/UpdateRemoveEmploymentFormSpec.scala (hunk 2 above)
Comment (Muhammad-Ali-Ahmad) [reply test/uk/gov/hmrc/tai/forms/employments/UpdateRemoveEmploymentFormSpec.scala]: done

File: test/views/html/employments/UpdateRemoveEmploymentDecisionViewSpec.scala (hunk 3 above)
Comment (Muhammad-Ali-Ahmad) [reply test/views/html/employments/UpdateRemoveEmploymentDecisionViewSpec.scala]: done

**Review by struong — APPROVED** [review]

**Review by struong — APPROVED** [review]

**Pull request merged**